pluwum/react-shoppinglist
# #152709501 User should able to login with username and password

#### What does this PR do?
The PR adds crude login capability to the application

#### How should this be manually tested?
In the working browser application , create an account then navigate to "/". type email and pasword and submit input. User should be logged into the application

#### What are the relevant pivotal tracker stories?
152709501
